README.md

```diff
@@ -44,16 +44,17 @@ Read our paper for more details.
 
 ## 🚀 Setup <a name="setup"></a>
 1. [Install Docker](https://docs.docker.com/engine/install/), then start Docker locally.
-2. [Install Miniconda](https://docs.anaconda.com/free/miniconda/miniconda-install/), then create the `swe-agent` environment with `conda env create -f environment.yml`
-3. Activate using `conda activate swe-agent`.
-4. Run `./setup.sh` to create the `swe-agent` docker image.
-5. Create a `keys.cfg` file at the root of this repository and fill in the following:
+2. Clone the repository with `git clone https://github.com/princeton-nlp/SWE-agent.git && cd SWE-agent`
+3. [Install Miniconda](https://docs.anaconda.com/free/miniconda/miniconda-install/), then create the `swe-agent` environment with `conda env create -f environment.yml`
```

Comment:
Maybe link to the command line install? Seems to be the easier way: https://docs.anaconda.com/free/miniconda/#quick-command-line-install

Reply:
```suggestion
3. [Install Miniconda](https://docs.anaconda.com/free/miniconda/#quick-command-line-install), then create the `swe-agent` environment with `conda env create -f environment.yml`
```